packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -320,4 +317,9 @@ export class ErrorBars
     private errorBarFactory(): ErrorBarNode {
         return new ErrorBarNode();
     }
+
+    private getWhiskerProperties() {
+        const { stroke, strokeWidth, visible, strokeOpacity, lineDash } = this;
```

Comment:
The `lineDashOffset` value also needs to be returned.

nit: It might be a good idea to explicitly specify the `ErrorBarWhiskerTheme` return type for `getWhiskerProperties()` to hopefully catch some of these errors at compiler-time.